ipatests/test_integration/test_sssd.py

```diff
@@ -403,3 +407,173 @@ def test_trustdomain_disable_disables_subdomain(self):
                 assert 'no such user' in res.stderr_text
         # verify the user can be retrieved after re-enabling trustdomain
         self.master.run_command(['id', user])
+
+
+class TestNestedMembers(IntegrationTest):
+    num_clients = 1
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master)
+        tasks.install_client(cls.master, cls.clients[0])
+
+    def test_nested_group_members(self):
+        """Nested group memberships should be honoured
+
+        "child2" should be a child of "child1"
+        so that parent child relationship is as follows:
+        "parent"->"child1"->"child2"
```

Comment:
I find the naming confusing.  "parent" is a member (subset) of "child1", and "child1" is a member (subset) of "child2".
It's a nit-pick, but even just calling them 'A' 'B' and 'C' would help, because those names don't have contradictory meanings.

Reply:
that's primary reason why I had used a,b,c but there was another suggestion to use grandparent, parent and child from @mrizwan93 

> It would be nice to have group name like `parent` `child1` and `child2`.
> OR
> `grandparent` > `parent` > `child` (a > b >c).
> 
> It would make easy to understand who is who's subgroup.

but I guess that would be also confusing.

Reply:
Please don't mark a comment as resolved unless you either apply the proposed change or argument why you disagree. As @frasertweedale I find the naming confusing, and think that subgroup / group / supergroup or even A / B / C would be better.

Reply:
changed the group names to a,b,c as requested.